packages/plugin-ext/src/main/browser/plugin-icon-theme-service.ts

```diff
@@ -390,6 +390,12 @@ export class PluginIconTheme extends PluginIconThemeDefinition implements IconTh
     ): void {
         if (associations.folder) {
             accept(associations.folder, this.folderIcon);
+            if (!('folderExpanded' in associations)) {
```

Comment:
Suggestion: Maybe use the following, as it is more type safe?
```suggestion
            if (associations.folderExpanded === undefined) {
```

Reply:
Right, that was my thought as well.